documentation/userguide.md

```diff
@@ -307,6 +307,25 @@ match:
       fileUpload: http://localserver.localdomain:8080
 ```
 
+## Example: Azure File Storage
+
+**UploadDaemon.yaml:**
+
+  To upload traces to an Azure File Storage first obtain the [connection string](https://docs.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string) for your storage account.
```

Comment:
please don't use inline links. Use named references and put the link at the bottom of the file. That makes reading the markdown easier while editing